CMakeLists.txt

```diff
@@ -139,6 +139,18 @@ if(CMAKE_COMPILER_IS_MSVC)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /WX")
 endif(CMAKE_COMPILER_IS_MSVC)
 
+if(MSVC)
+    option(BUILD_WITH_STATIC_RUNTIME "Build the libraries with /MT compiler flag" OFF)
```

Comment:
I'm not sure the name of the option is what we want. It's a bit long, and this is a Microsoft specific feature. Maybe something like `MSVC_STATIC_RUNTIME`?